### ISSUE TITLE
 the mod does not work on Quest 3

### ISSUE THREAD

**Issue by VladAlon** [convexvr/sims4-vr#11]
Unfortunately, the mod does not work on Quest 3. I did everything according to the instructions, but the mod won't start. I press the B button during the game and nothing happens. The debug_log.txt file doesn't appear in the folder either. Please help me to solve the problem. Other mods in the game work

**Comment by conkydong** [comment]
Same here for me. Anyone knows a fix?

**Comment by HighSodium** [comment]
Use Virtual Desktop. Not airlink or usb.

**Comment by conkydong** [comment]
I did use Virtual Desktop, it's not working.

**Comment by convexvr** [comment]
@conkydong this is probably due to the nex dx 11 version of the sims. I will close this issue now. Until we have debug symbols or some AI thing that can help with decompilng the executable, development on this mod is on pause.

**Comment by Laddcus** [comment]
Tried launching Dx9 ver of sims 4 with steam Vr on quest 3 and got the same issue. Cant enter 1st person with 'B' button on controller or Shift +Tab on keyboard.

**Comment by convexvr** [comment]
Maxis changed part of the render interface as they introduced dx11, the dx9 version wont work either.

**Comment by Laddcus** [comment]
Damn ok. honestly just getting a taste of it in Virtual Desktop with the screen up close in 1st person mode was elite. Having VR supported would be a game changer.

**Comment by HighSodium** [comment]
I ended up getting it working on the Index by launching in DX9 mode. DX11 would not work at all.

When I went into the VR mode, my dude's head disappeared and I was able to move around with keys/mouse, which means the addon was hooking. Although the scale of things was waaaay off, zoomed in, and it didn't center around my sim's neck. If my sim moved, the camera wouldn't follow him and he would just be headless across the room. Also the keys for the controller didn't work as intended (which was to be expected).
